rules/windows/defense_evasion_windows_firewall_profile_disabled.toml

```diff
@@ -0,0 +1,57 @@
+[metadata]
```

Comment:
File to be renamed:
defense_evasion_windows_firewall_profile_disabled.toml -> defense_evasion_powershell_windows_firewall_disabled.toml

Reply:
Still needs to be renamed